src/lib/kube-enclave.ts

```diff
@@ -72,6 +76,78 @@ class KubernetesEnclave extends Enclave<KubernetesApiJobsResponse> implements IE
             throw new Error(errMsg)
         }
     }
+    async cleanup(): Promise<void> {
+        console.log('Cleaning up the enclave!')
+        const jobsInEnclave = await this.getAllStudiesInEnclave()
+        const containers = ((await k8sApiCall(undefined, 'pods', 'GET')) as KubernetesApiJobsResponse).items.flatMap(
+            (j) => j as KubernetesPod,
+        )
+        jobsInEnclave.forEach(async (job) => {
```

Comment:
It seems like `cleanup()` does not actually wait for all the jobs to finish cleaning up because of this `.forEach()`. Is that the intent or maybe I'm misreading something?